common/testData/localeIdentifiers/localeCanonicalization.txt

```diff
@@ -35,8 +35,8 @@ hy_arevela	;	hy
 hy_arevmda	;	hyw
 hy_arevmda_arevela	;	hyw
 hye_arevmda	;	hyw
-no_bokmal_nynorsk	;	nb
-no_nynorsk_bokmal	;	nb
+no_bokmal_nynorsk	;	no
+no_nynorsk_bokmal	;	no
```

Comment:
@macchiati I think this change in lines 38-39 is wrong (and so does the code in ICU4C, ICU4J). I filed https://unicode-org.atlassian.net/browse/CLDR-16476 about this.